pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_crd_reconcile.go

```diff
@@ -109,6 +122,13 @@ func (c *crdReconcile) clean(ctx context.Context, cm *operatorapiv1.ClusterManag
 		return cm, reconcileContinue, nil
 	}
 
+	// CRD resource files to clean
+	hubDeployCRDResources := hubCRDResourceFiles
+
+	// Include the ClusterProfile CRD in case it was installed, since the CRD will not be
```

Comment:
the comment said the crd will not be removed, so why should we clean up it here?

Reply:
The ClusterProfile crd is not removed when featuregates disable, but will be cleaned when ClusterManager is deleting, the behavior of disable and clean (except install) are the same as other crds.  
But this is a good question since ClusterProfile CRD might be used by other project, might need to leave it.

Reply:
yes let's just leave it.

Reply:
fixed.